.github/workflows/python_release.yml

```diff
@@ -35,47 +35,23 @@ jobs:
       fail-fast: false
       matrix:
         target: [x86_64-apple-darwin, aarch64-apple-darwin]
-    runs-on: macos-11
+    runs-on: macos-12
     steps:
       - uses: actions/checkout@v3
 
-      # We use extra recent Cargo.toml syntax, so we need at least Rust 1.71.0
-      - name: Install newer rust
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: default
-          toolchain: stable
-          override: true
-
       - name: Publish to pypi (without sdist)
         uses: messense/maturin-action@v1
         env:
           MATURIN_PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
         with:
           target: ${{ matrix.target }}
           command: publish
-          args: -m python/Cargo.toml --no-sdist ${{ env.FEATURES_FLAG }}
-
-  release-pypi-mac-universal2:
```

Comment:
I don't think anyone really has need for this.

Reply:
What are those feature flags setting there?

Reply:
You mean this?

https://github.com/delta-io/delta-rs/blob/573cbd139a5607d72d12fc0480884d8a517275b3/.github/workflows/python_release.yml#L12-L14